bin/healthcheck_processor_notification.sh

```diff
@@ -23,27 +23,29 @@ if [ "${TOKEN_NOTIFICATION_ENABLED}" = 1 ]; then
   PROC_LIST="${PROC_LIST} batch/processor_Notifications_Token.py"
 fi
 
-if [ "${BOND_TOKEN_ENABLED}" = 1 ]; then
-  if [ ! -z "${IBET_SB_EXCHANGE_CONTRACT_ADDRESS}" ]; then
-    PROC_LIST="${PROC_LIST} batch/processor_Notifications_Bond_Exchange.py"
+if [ "${EXCHANGE_NOTIFICATION_ENABLED}" = 1 ]; then
```

Comment:
[MEMO]
Before:

 - `batch/processor_Notifications_{}_Exchange.py` run automatically.

After:

 - `batch/processor_Notifications_{}_Exchange.py` doesn't run unless `EXCHANGE_NOTIFICATION_ENABLED` is not set 1.